bundles/binding/org.openhab.binding.ebus/src/main/java/org/openhab/binding/ebus/internal/connection/EBusSerialConnector.java

```diff
@@ -0,0 +1,95 @@
+/**
+* Copyright (c) 2010-2015, openHAB.org and others.
+*
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Eclipse Public License v1.0
+* which accompanies this distribution, and is available at
+* http://www.eclipse.org/legal/epl-v10.html
+*/
+package org.openhab.binding.ebus.internal.connection;
+
+import gnu.io.CommPortIdentifier;
+import gnu.io.NoSuchPortException;
+import gnu.io.PortInUseException;
+import gnu.io.SerialPort;
+import gnu.io.UnsupportedCommOperationException;
+
+import java.io.IOException;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+* @author Christian Sowada
+* @since 1.7.0
+*/
+public class EBusSerialConnector extends AbstractEBusConnector {
+
+	private static final Logger logger = LoggerFactory
+			.getLogger(EBusSerialConnector.class);
+	
+	/** The serial object */
+	private SerialPort serialPort;
+	
+	/** The serial port name */
+	private String port;
+
+	/**
+	 * @param port
+	 */
+	public EBusSerialConnector(String port) {
+		this.port = port;
+	}
+
+	/* (non-Javadoc)
+	 * @see org.openhab.binding.ebus.connection.AbstractEBusConnector#connect()
+	 */
+	@Override
+	public boolean connect() throws IOException {
+		try {
+			final CommPortIdentifier portIdentifier = CommPortIdentifier.getPortIdentifier(port);
+
+			if(portIdentifier != null) {
+				serialPort = (SerialPort) portIdentifier.open("openhab-ebus", 5000);
+				serialPort.setSerialPortParams(2400, SerialPort.DATABITS_8,
+						SerialPort.STOPBITS_1, SerialPort.PARITY_NONE);

```

Comment:
can you confirm that all heating systems connect using the same speed? If not please make (at least the speed) configurable